docs/guide/getting-started.md

```diff
@@ -263,7 +262,7 @@ Completions are placed in a ZSH friendly location, but [ZSH must be configured t
 Add `asdf.nu` to your `~/.config/nushell/config.nu` with:
 
 ```shell
-"\nlet-env ASDF_NU_DIR = ($env.HOME | path join '.asdf')\n source " + ($env.HOME | path join '.asdf/asdf.nu') | save --append $nu.config-path 
```

Comment:
I agree we should use Shell best-practices in the documentation code snippets, however, I am not so sure if this is considered best practice for Nushell. Also iffy about Fish.

Reply:
Yeah the non-POSIX shells could have better snippets. Since I don't deal much with those shells I didn't touch their code snippets (besides the trailing whitespace)

Reply:
Oh I see! 😅 silly me, I didn't read the diff properly. Thanks!